test/e2e/tests/zone_aware_routing.go

```diff
@@ -28,6 +28,11 @@ var ZoneAwareRoutingTest = suite.ConformanceTest{
 	Description: "Resource with Zone Aware Routing enabled",
 	Manifests:   []string{"testdata/zone-aware-routing.yaml"},
 	Test: func(t *testing.T, suite *suite.ConformanceTestSuite) {
+		// TODO: make this work with gateway namespace mode
```

Comment:
cc @dprotaso

Reply:
did you mean to tag me?

Reply:
can you take a look why this failed on gateway namespace mode later?

Reply:
You might be tagging the wrong person - I'm not aware of zone aware routing etc.

Reply:
it is @jukie

Reply:
by default, topology annotations are only added for pods in the controller ns, because of how the mutation webhook filtering works, logic will need to be tweaked for gateway namespace mode, imo this shouldnt block v1.4 release

Reply:
@dprotaso sorry for the wrong tagging.